release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -579,6 +579,17 @@ If the following conditions are true, the feature is enabled by default:
 [id="ocp-4-15-oci"]
 === Oracle(R) Cloud Infrastructure
 
+[id="ocp-4-15-installation-oci-agent-based-installer-vm"]
+==== Using the Agent-based Installer to install a cluster on {oci} (Technology Preview)
+
+You can use the Agent-based Installer to install a cluster on {oci-first}, so that you can run cluster workloads on infrastructure that supports dedicated, hybrid, public, and multiple cloud environments.
+
+The Agent-based installer provides the ease of use of the Assisted Installation service, but with the capability to install a cluster in either a connected or disconnected environment.
+
+{oci} provides services that can meet your regulatory compliance, performance, and cost-effectiveness needs. {oci} supports 64-bit `x86` instances and `64-bit` ARM instances.
```

Comment:
I might be wrong here. But I think it could be consistent.

```suggestion
{oci} provides services that can meet your regulatory compliance, performance, and cost-effectiveness needs. {oci} supports 64-bit `x86` instances and 64-bit ARM instances.
```